www/vercel.json

```diff
@@ -15,6 +15,11 @@
       "destination": "/docs/react-queries",
       "permanent": true
     },
+    {
+      "source": "/docs/invalidateQueries",
```

Comment:
```suggestion
      "source": "/docs/invalidateQuery",
```
?

Reply:
It's renamed to `invalidateQueries` to adhere to react-query

Reply:
OH!

good catch!